.gitmodules

```diff
@@ -959,6 +959,9 @@
 [submodule "vendor/grammars/nesC"]
 	path = vendor/grammars/nesC
 	url = https://github.com/cdwilson/nesC.tmbundle
+[submodule "vendor/grammars/nickel"]
+	path = vendor/grammars/nickel
```

Comment:
the TextMate grammar is a couple of directories deep: https://github.com/tweag/nickel/tree/master/lsp/vscode-extension

I don't know if that's going to cause issues

Reply:
You successfully managed to add the grammar using `script/add-grammar` so this isn't a problem.